compiler/AST/AstToText.cpp

```diff
@@ -1372,3 +1389,36 @@ bool AstToText::isMtArg(CallExpr* expr, bool expectThis) const
   return retval;
 }
 
+void AstToText::appendEnumDecl(EnumType* et) {
+  mText += et->symbol->name;
+
+  appendEnumConstants(et);
+}
+
+void AstToText::appendEnumConstants(EnumType* et) {
+  int count = et->constants.length;
+  bool first = true;
+
+  if (count > 0) {
+    mText += " { ";
+
+    for_alist(constant, et->constants) {
+      if (DefExpr* de = toDefExpr(constant)) {
+        if (!first) {
+          mText += ", ";
+        } else {
+          first = false;
+        }
+        appendExpr(de, false);
+      } else {
+        // LYDIA 2015/04/06 Debugging support.  Means that the list of enum
+        // constants contained something that wasn't a DefExpr, which is
+        // unexpected
+        mText += " appendEnumConstants.00";
```

Comment:
To make the case a little stronger for INT_ASSERT()/INT_FATAL(), I think it would be easy to miss these extra bits of text in the chpldoc output (as an example). Using compiler errors will make the issues more obvious if/when they come up.

That said, it seems ok for that to be another effort to revise AstToText. Thanks!